### ISSUE TITLE
Deprecation warnings from create-application.py

### ISSUE THREAD

**Issue by motoom** [qooxdoo/qooxdoo#9490]
Running **create-application.py** from a freshly unzipped **qooxdoo-5.0.2-sdk.zip** gives the following deprecation warnings, with no clear hints how to resolve them (maybe this is not the responsibility of qooxdoo, but of npm? Or has it something to do with outdated dependencies?)

```
~/Prj/qooxdoo-5.0.2-sdk $python create-application.py -n WebApp

[...]

npm WARN deprecated coffee-script@1.3.3: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN WebApp@0.1.0 No license field.
```

**Comment by hkollmann** [comment]
This is fixed in the current SVN Version 6.0 alpha. npm is not needed for the python based toolchain. So you can just ignore the messsages

**Comment by motoom** [comment]
Thanks!
